tools/integration_tests/mounting/gcsfuse_test.go

```diff
@@ -84,12 +86,15 @@ func (t *GcsfuseTest) runGcsfuseWithEnv(args []string, env []string) (err error)
 	cmd := t.gcsfuseCommand(args, env)
 
 	// Run.
+	start := time.Now()
 	output, err := cmd.CombinedOutput()
 	if err != nil {
 		err = fmt.Errorf("error %w running gcsfuse; output:\n%s", err, output)
 		return
 	}
-
+	if mountTime := time.Since(start).Seconds(); mountTime > 2.5 {
```

Comment:
Can we keep 2.5 seconds as constant? Something like expectedMountTime?

Reply:
Updated.